newIDE/app/src/ObjectEditor/Editors/SpriteEditor/DirectionTools.js

```diff
@@ -86,8 +78,8 @@ const DirectionTools = ({
 
     if (newTimeIsValid) {
       direction.setTimeBetweenFrames(newTimeBetweenFrames);
```

Comment:
Do you need to apply `time.toFixed(6)` like it was done before when calling `formatTime()?`

Reply:
I have the same answer as for the other comment. I'd rather not used toFixed on those values